cime_config/machines/config_machines.xml

```diff
@@ -3251,6 +3251,113 @@
     </environment_variables>
   </machine>
 
+  <machine MACH="chicoma-cpu">
+    <DESC>Chicoma CPU-only nodes at LANL IC. Each node has 2 AMD EPYC 7H12 64-Core (Milan) 512GB</DESC>
+    <NODENAME_REGEX>ch-fe*</NODENAME_REGEX>
+    <OS>Linux</OS>
+    <COMPILERS>gnu,nvidia,aocc,amdclang</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <CIME_OUTPUT_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/scratch/chicoma-cpu</CIME_OUTPUT_ROOT>
```

Comment:
```suggestion
    <CIME_OUTPUT_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/e3sm_scratch/chicoma-cpu</CIME_OUTPUT_ROOT>
```
For now, I think the `E3SM/scratch` and `E3SM/input_data` directories are okay but once we move to a common input directory, it might be nice to be more consistent with other E3SM machines.